internal/cli/clients/epinio_api.go

```diff
@@ -24,10 +25,32 @@ type EpinioAPIClient struct {
 var epinioClientMemo *EpinioAPIClient
 
 func GetEpinioAPIClient(ctx context.Context) (*EpinioAPIClient, error) {
+	// Check for information cached in memory, and return if such is found
 	if epinioClientMemo != nil {
 		return epinioClientMemo, nil
 	}
 
+	// Check for information cached in the Epinio configuration,
+	// and return if such is found. Cache into memory as well.
+	configConfig, err := config.Load()
```

Comment:
nit: Why not just `cfg`?

> Variable names in Go should be short rather than long. This is especially true for local variables with limited scope. Prefer c to lineCount. Prefer i to sliceIndex.

https://github.com/golang/go/wiki/CodeReviewComments#variable-names

Reply:
Fixed with commit 53b826e55dfc618ab4406227517b4deb030a2bd5